pkg/storaged/fsys-tab.jsx

```diff
@@ -381,20 +410,25 @@
             opts.push("ro");
         if (opt_never_auto)
             opts.push("x-cockpit-never-auto");
+        if (opt_nofail)
+            opts.push("nofail");
+        if (opt_netdev)
+            opts.push("_netdev");
         if (forced_options)
             opts = opts.concat(forced_options);
         if (extra_options)
             opts = opts.concat(extra_options);
-        return maybe_set_crypto_options(null, false).then(() => maybe_update_config(old_dir, unparse_options(opts)));
+        return (maybe_set_crypto_options(null, false, null, null)
+                .then(() => maybe_update_config(old_dir, unparse_options(opts))));
     }
 
     let passphrase_type;
 
-    function maybe_set_crypto_options(readonly, auto) {
+    function maybe_set_crypto_options(readonly, auto, nofail, netdev) {
         if (client.blocks_crypto[block.path])
-            return set_crypto_options(block, readonly, auto);
+            return set_crypto_options(block, readonly, auto, nofail, netdev);
         else if (client.blocks_crypto[block.CryptoBackingDevice])
-            return set_crypto_options(client.blocks[block.CryptoBackingDevice], readonly, auto);
+            return set_crypto_options(client.blocks[block.CryptoBackingDevice], readonly, auto, nofail, netdev);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-142402-f4c856ed-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)